medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -29,10 +30,65 @@
 from ..indexer_exceptions import (IndexerError, IndexerException, IndexerShowIncomplete, IndexerShowNotFound,
                                   IndexerShowNotFoundInLanguage, IndexerUnavailable)
 from ..indexer_ui import BaseUI, ConsoleUI
+from medusa import ui
 
 
 logger = logging.getLogger(__name__)
 
+API_BASE_TVDB = 'https://api.thetvdb.com'
+API_BASE_URL_FALLBACK = 'https://tvdb2.plex.tv'
+
+
+def plex_fallback(func):
+    """Fallback to plex if tvdb fails to connect."""
+    def inner(*args, **kwargs):
+        config = args[0].config['session'].fallback_config
+
+        def fallback_notification():
+            ui.notifications.error('Tvdb2.plex.tv fallback',
```

Comment:
I'd also add a warning log. UI notification require the user to be on the Web UI?